Mathlib/Algebra/Algebra/Defs.lean

```diff
@@ -98,16 +98,17 @@ section Prio
 See the implementation notes in this file for discussion of the details of this definition.
 -/
 -- Porting note (https://github.com/leanprover-community/mathlib4/issues/5171): unsupported @[nolint has_nonempty_instance]
-class Algebra (R : Type u) (A : Type v) [CommSemiring R] [Semiring A] extends SMul R A,
-  R →+* A where
-  commutes' : ∀ r x, toRingHom r * x = x * toRingHom r
-  smul_def' : ∀ r x, r • x = toRingHom r * x
+class Algebra (R : Type u) (A : Type v) [CommSemiring R] [Semiring A] extends SMul R A where
+  /-- Embedding `R →+* A` given by `Algebra` structure. -/
```

Comment:
This should include a remark that `algebraMap` in the root namespace should be used instead.